engine/compiler/util.go

```diff
@@ -190,6 +188,48 @@ func isRestrictedVariable(env map[string]*manifest.Variable) bool {
 	return false
 }
 
+// Upper value for cpu and memory requests. It is same as stage level
+// cpu & memory requests.
+func getStepUpperRequestVal(r resource.StageResources) ResourceObject {
+	return ResourceObject{
+		CPU:    r.Requests.CPU,
+		Memory: int64(r.Requests.Memory),
+	}
+}
+
+// Lower value for cpu and memory requests.
+func getStepLowerRequestVal() ResourceObject {
+	r := ResourceObject{
+		CPU:    0,
+		Memory: 0,
+	}
+
+	cpuStr := os.Getenv("DRONE_RESOURCE_MIN_REQUEST_CPU")
```

Comment:
we collect all environment variables here:
https://github.com/drone-runners/drone-runner-kube/blob/master/command/daemon/config.go

and then we pass variables to the compiler through this struct:
https://github.com/drone-runners/drone-runner-kube/blob/master/engine/compiler/compiler.go#L64

Reply:
Fixed